Library/ViewModels/ManageViewPledgeViewModelTests.swift

```diff
@@ -0,0 +1,83 @@
+import Foundation
+@testable import KsApi
+@testable import Library
+import Prelude
+import ReactiveExtensions_TestHelpers
+import ReactiveSwift
+import XCTest
+
+internal final class ManageViewPledgeViewModelTests: TestCase {
+  private let vm = ManageViewPledgeViewModel()
+
+  private let configurePaymentMethodView = TestObserver<Project, Never>()
+  private let configurePledgeSummaryView = TestObserver<Project, Never>()
+  private let configureRewardSummaryView = TestObserver<Reward, Never>()
+  private let title = TestObserver<String, Never>()
+
+  override func setUp() {
+    super.setUp()
+    self.vm.outputs.title.observe(self.title.observer)
+    self.vm.outputs.configurePaymentMethodView
+      .observe(self.configurePaymentMethodView.observer)
+    self.vm.outputs.configurePledgeSummaryView
+      .observe(self.configurePledgeSummaryView.observer)
+    self.vm.outputs.configureRewardSummaryView
+      .observe(self.configureRewardSummaryView.observer)
+  }
+
+  func testNavigationBarTitle_LiveProject() {
+    self.title.assertDidNotEmitValue()
+
+    let project = Project.template
+    self.vm.inputs.configureWith(project, reward: .template)
+
+    self.vm.inputs.viewDidLoad()
+
+    self.title.assertValue(Strings.Manage_your_pledge())
```

Comment:
Can we use the actual values of these strings in here? It's better to be more explicit about what you're testing against:

```
self.title.assertValue("Manage your pledge")
```